sdk/formrecognizer/Azure.AI.FormRecognizer/src/BoundingBox.cs

```diff
@@ -16,26 +17,28 @@ namespace Azure.AI.FormRecognizer.Models
     /// </summary>
     public readonly struct BoundingBox
     {
+        private readonly PointF[] _points;
```

Comment:
We **cannot** override the public parameterless constructor that structs have, so `Points` was forcibly being set to null in a default `BoundingBox`, whether we liked it or not. We also could not update it later on, since this struct is `readonly`.

The solution I thought about was using a private `_points` field to take that role, and accessing it via the internal `Points` property, which will return an empty array if `_points` is null.